install_config/monitoring/configuring-etcd-monitoring.adoc

```diff
@@ -185,3 +185,13 @@ $ oc apply -f etcd-cert-secret.yaml
 . Now that you have configured authentication, visit the *Targets* page of the web interface again. Verify that `etcd` is now being correctly monitored. It might take several minutes for changes to take effect.
 +
 image::etcd-monitoring-working.png[]
+
+. If you want `etcd` monitoring to be automatically updated when you update {product-title}, set this variable in the Ansible inventory file to `true`:
++
+`openshift_cluster_monitoring_operator_etcd_enabled`
++
+If you run `etcd` on separate hosts, specify the nodes by IP addresses using this Ansible variable:
++
+`openshift_cluster_monitoring_operator_etcd_hosts`
```

Comment:
Same comment about the above, maybe with an example of how the IP addresses look

```
----
openshift_cluster_monitoring_operator_etcd_hosts= // i don't know how they need to be... comma separated? in quotes?
----
```